tests/lit-tests/ispc-intrinsics.ispc

```diff
@@ -0,0 +1,78 @@
+// RUN: %{ispc} %s --nowrap --target=host --emit-llvm-text --enable-llvm-intrinsics -o - 2>&1 | FileCheck %s
+
+#define unmangled __attribute__((unmangled))
+#define ATTRS unmasked unmangled
+
+#if TARGET_WIDTH == 2
+#define DOUBLE_WIDTH 4
+#elif TARGET_WIDTH == 4
+#define DOUBLE_WIDTH 8
+#elif TARGET_WIDTH == 8
+#define DOUBLE_WIDTH 16
+#elif TARGET_WIDTH == 16
+#define DOUBLE_WIDTH 32
+#elif TARGET_WIDTH == 32
+#define DOUBLE_WIDTH 64
+#elif TARGET_WIDTH == 64
+#define DOUBLE_WIDTH 128
+#else
+#error Unknown value of TARGET_WIDTH
+#endif
+
+// CHECK: define void @mask_type___(<[[WIDTH:.*]] x [[MASK:.*]]> %__mask)
+void mask_type() {}
+
+// CHECK-LABEL: @atomicrmw
+// CHECK-DAG:   %calltmp = tail call double @llvm.ispc.atomicrmw.xchg.seq_cst.f64([[PTR:(ptr|i8\*)]] %ptr, double %val)
+ATTRS uniform double atomicrmw(uniform int8 *uniform ptr, uniform double val) {
+    return @llvm.ispc.atomicrmw.xchg.seq_cst(ptr, val);
+}
+
+// CHECK-LABEL: @bitcast
+// CHECK-DAG:   %calltmp = tail call half @llvm.ispc.bitcast.i16.f16(i16 %x, half 0xH0000)
+ATTRS uniform float16 bitcast(uniform int16 x) {
+    return @llvm.ispc.bitcast(x, (uniform float16)0);
+}
+
+// TODO!
```

Comment:
Missed TODO here?

Reply:
Indeed, `concat` was needed in one more piece of code to be supported.